SPECS/pam_krb5/pam_krb5.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        A Pluggable Authentication Module for Kerberos 5
+Name:           pam_krb5
+Version:        4.11
+Release:        2%{?dist}
+License:        MIT AND BSD
+Group:          System/Libraries
+URL:            https://github.com/rra/pam-krb5
+Source0:        %{url}/archive/refs/tags/upstream/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0001:      0001-Drop-module-long-test.patch
+BuildRequires:  byacc
+BuildRequires:  flex
+BuildRequires:  gcc
+BuildRequires:  krb5-devel
+BuildRequires:  pam-devel
+BuildRequires:  perl(Test::Pod)
+# for testing
```

Comment:
Can we gate check-time requirements using the proper macros?

Reply:
Fixed